packages/model/addon/index.ts

```diff
@@ -35,7 +35,7 @@
 
   @module @ember-data/model
   @main @ember-data/model
-  @class model_package_named_exports
+  @class Model
```

Comment:
I think we fully remove `class` here if we aren't using `model_package_named_exports` to create the fake class

Reply:
Keeping class here as "Model" creates a useful link to Model class.